docs/pages/versions/unversioned/sdk/asset.mdx

```diff
@@ -9,13 +9,55 @@ platforms: ['android', 'ios', 'tvos', 'web']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
+import { ConfigPluginExample, ConfigPluginProperties } from '~/components/plugins/ConfigSection';
 
-**`expo-asset`** provides an interface to Expo's asset system. An asset is any file that lives alongside the source code of your app that the app needs at runtime. Examples include images, fonts, and sounds. Expo's asset system integrates with React Native's, so that you can refer to files with `require('path/to/file')`. This is how you refer to static image files in React Native for use in an `Image` component, for example. Check out React Native's [documentation on static image resources](https://reactnative.dev/docs/images#static-image-resources) for more information. This method of referring to static image resources works out of the box with Expo.
+`expo-asset` provides an interface to Expo's asset system. An asset is any file that lives alongside the source code of your app that the app needs at runtime. Examples include images, fonts, and sounds. Expo's asset system integrates with React Native's, so that you can refer to files with `require('path/to/file')`. This is how you refer to static image files in React Native for use in an `Image` component, for example. Check out React Native's [documentation on static image resources](https://reactnative.dev/docs/images#static-image-resources) for more information. This method of referring to static image resources works out of the box with Expo.
 
 ## Installation
 
 <APIInstallSection />
 
+## Configuration in app.json/app.config.js
+
+You can configure `expo-asset` using its built-in [config plugin](/config-plugins/introduction/) if you use config plugins in your project ([EAS Build](/build/introduction) or `npx expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
+
+<ConfigPluginExample>
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-asset",
+        {
+          "assets": ["path/to/file.png", "path/to/directory"]
+        }
+      ]
+    ]
+  }
+}
+```
+
+</ConfigPluginExample>
+
+<ConfigPluginProperties
+  properties={[
+    {
+      name: 'assets',
+      description: [
+        'An array of asset files or directories to link to the native project. The paths should be relative to the project root so that the file names, whether specified directly or using a directory, will become the resource names.',
+        '',
+        'Supported file types:',
+        '* Images: `.png`, `.jpg`, `.gif`',
+        '* Media: `.mp4`, `.mp3`, `.lottie`',
+        '* SQLite database files: `.db`'
```

Comment:
i think we should consolidate metro [assetExts](https://metrobundler.dev/docs/configuration/#assetexts) for the extra file types. currently .lottie and .db here are not default. but .lottie is still not default though.
would be good to document some necessary change if people want to include extra file types.

for a reference of default assetExts (it may change from time to time)
- metro's default [assetExts](https://github.com/facebook/metro/blob/c3aec551139faace766031f65569e073a59002b8/packages/metro-config/src/defaults/defaults.js#L18-L52)
- our extra [assetExts](https://github.com/expo/expo/blob/d091e0a1933c39743c620dbffbb115247f2c8b0c/packages/%40expo/metro-config/src/ExpoMetroConfig.ts#L188-L192)

Reply:
@Kudo, I see the that the built-in support for `.db` and updatded instructions in SQLite API reference.

I decided to add a note which links to the instruction from SQLite API reference section here because the config plugin is the closest place where someone can find this information:

![CleanShot 2024-04-20 at 01 58 52@2x](https://github.com/expo/expo/assets/10234615/ef0959fe-cc1a-4f59-b9e4-50a43cb0e1f0)

Should we refrain mentioning `.lottie` here? @alanjhughes mentioned that it kind of works out fo the box when he tested.